### PR TITLE
ramips-mt7620: remove "broken" status for ASUS RT-AC51U

### PR DESCRIPTION
**Pull request by rabbit83ka**
The reasons for "broken" status (no 5 GHz WLAN, LEDs not working) are [fixed in OpenWRT.](https://openwrt.org/toh/asus/rt-ac51u) Only 5 GHz-LED isn't working at all.

- [x] Must be flashable from vendor firmware
  - [ ] Web interface
  - [x] TFTP
  - [x] Other: ASUS Firmware Restoration Tool, see [OpenWRT-Wiki](https://openwrt.org/toh/asus/rt-ac51u)
- [x] Must support upgrade mechanism
  - [x] Must have working sysupgrade
    - [x] Must keep/forget configuration (`sysupgrade [-n]`, `firstboot`)
  - [x] Gluon profile name matches autoupdater image name:  is asus-rt-ac51u
        (`lua -e 'print(require("platform_info").get_image_name())'`)
- [x] Reset/WPS/... button must return device into config mode
- [x] Primary MAC address should match address on device label (or packaging)
      (https://gluon.readthedocs.io/en/latest/dev/hardware.html#notes)
  - When re-adding a device that was supported by an earlier version of Gluon, a
    factory reset must be performed before checking the primary MAC address, as
    the setting from the old version is not reset otherwise.
- Wired network
  - [x] should support all network ports on the device
  - [x] must have correct port assignment (WAN/LAN)
    - On devices supplied via PoE, there is usually no explicit WAN/LAN labeling on the hardware.
      The PoE input should be the WAN port in this case.
- Wireless network (if applicable)
  - [x] Association with AP must be possible on all radios
  - [x] Association with 802.11s mesh must work on all radios 
  - [x] AP+mesh mode must work in parallel on all radios
- LED mapping
  - Power/system LED
    - [x] Lit while the device is on
    - [x] Should display config mode blink sequence 
          (https://gluon.readthedocs.io/en/latest/features/configmode.html)
  - Radio LEDs --> Only 2,4. GHz LED is working, 5 GHz doen't work
    - [x] Should map to their respective radio
    - [x] Should show activity
  - Switch port LEDs
    - [x] Should map to their respective port (or switch, if only one led present) 
    - [x] Should show link state and activity
- Outdoor devices only: (not applicable, indoor device)

### CODE REVIEW

**Comment by rotanid** [comment]
@rabbit83ka please rebase your pull request so there are only one or at most two commits. the commit message could say the same as your PR title

maybe someone has an idea on how to fix the 5ghz LED?

**Comment by rabbit83ka** [comment]
@rotanid I rebased the the pull request, now it's one commit. I hope i did it right, was the first time..

File: docs/user/supported_devices.rst [hunk 1]
```diff
@@ -275,6 +279,7 @@ ramips-mt7621
 * ASUS
 
   - RT-AC57U
+
```
Comment (rotanid) [inline docs/user/supported_devices.rst:282]: why did you add this empty line?

Comment (rabbit83ka) [reply docs/user/supported_devices.rst:282]: I think accidentally. In a first version, I've put the RT-AC51U under the RT-AC57U. Then I obviously deleted only the content, not the line.

Comment (rabbit83ka) [reply docs/user/supported_devices.rst]: I removed the unnecessary empty line.
